<launch>
	
	<node name="checkers_manager" pkg="irp6_checkers" type="checkers_manager" output="screen"/>

	<node name="irp6_control" pkg="irp6_checkers" type="IRPOSControl.py" output="screen"/>
	
</launch>
